guides/source/action_mailbox_basics.md

```diff
@@ -43,6 +43,10 @@ $ bin/rails db:migrate
 
 ## Configuration
 
+By default, ActionMailbox will use ActiveStorage service configured (`config.active_storage.service`).
```

Comment:
add to https://guides.rubyonrails.org/configuring.html too

Reply:
in fact i'm not sure this needs to be in this guide, or specifically at the top of Configuring. I suspect this is something a minority of users will want to change.

Reply:
I agree that this is going to be of interest to a small group of users. That being said, I thought that burying it in the code and not documenting it isn't great either. What would be the best place for it?

Reply:
https://github.com/rails/rails/blob/main/guides/source/configuring.md

Reply:
Done